client/src/components/pages/Profile/staticData.js

```diff
@@ -0,0 +1,10 @@
+const fullTypeName = {
+  CBT: 'Cognitive Behavioural Therapy',
+  PD: 'Psychodynamic Therapy',
+  Hu: 'Humanistic psychotherapy',
+  In: 'ntegrative psychotherapy',
```

Comment:
```suggestion
  In: 'Integrative psychotherapy',
```
or make pull to master then push again